modules/clients.py

```diff
@@ -27,50 +27,41 @@ def done_at(self, time):
 
 class Clients(object):
 
-    def __init__(self, env, server, requests, process_time=0.00001, create_request_rate=0.01, max_requests=float("inf")):
+    def __init__(self, env, server, requests, process_time=0.00001, create_request_rate=0.01, max_requests=float("inf"), service_time=0.0035, memory=0.02):
         self.env = env
         self.server = server
         self.requests = requests
         self.process_time = process_time
         self.queue = simpy.Store(env)  # the queue of requests
         self.action = env.process(self.send_request())
-        self.create_request = env.process(self.create_request(create_request_rate, max_requests))
+        self.create_request = env.process(self.create_request(create_request_rate, max_requests, service_time, memory))
 
     def send_request(self):
         while True:
             if len(self.queue.items) > 0:  # check if there is any request to be processed
                 request = yield self.queue.get()  # get a request from store
-                print("At %.3f, CLIENTS a client sent a request" % self.env.now)
                 yield self.env.process(self.server.request_arrived(request))
 
             yield self.env.timeout(self.process_time) # wait for...
 
-    def create_request(self, create_request_rate, max_requests):
-        """ Create requests """
+    def create_request(self, create_request_rate, max_requests, service_time, memory):
```

Comment:
As it creates more than one request, could this be called create_request**s**?

Reply:
Yes! well change that too.

Reply:
Done, @danielfireman could you review?